package.json

```diff
@@ -47,6 +47,9 @@
     "crypto-js": "^4.0.0",
     "fastestsmallesttextencoderdecoder": "^1.0.22",
     "mqtt": "^4.2.8",
+    "tar": "^6.1.11",
+    "fs-extra": "^10.0.0",
+    "checksum": "^1.0.0",
```

Comment:
these are dependencies now for browser too, right? any way around that?

Reply:
It would be nice to dump these new dependencies in a future pass:

fs-extra: looks like it's being used for recursive copy. we could implement our own. looks like Node 16 added a more useful [fs.cpSync()](https://nodejs.org/api/fs.html#fs_fs_cpsync_src_dest_options), but we're currently supporting Node 10

checksum: don't think we need this? Here's an example using node's built in [crypto](https://nodejs.org/api/crypto.html#crypto_crypto_createhash_algorithm_options) module

Reply:
Yeah, I found that in Node 16 too. I'll try to implement it. But, I don't think we can avoid `tar`...